### ISSUE TITLE
breaking change in 0.8.13

### ISSUE THREAD

**Issue by rap2hpoutre** [xiaolin/react-image-gallery#381]
There is a breaking change in v0.8.13: 

https://github.com/xiaolin/react-image-gallery/pull/374/files#diff-eabaad0cc8751d0947757ba5349ae171R40

It crash with SSR (exemple with next.js' SSR): 

```
ReferenceError: window is not defined
    at new ImageGallery (/var/app/current/node_modules/react-image-gallery/build/image-gallery.js:367:5)
    at c (/var/app/current/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:33:323)
    at Sa (/var/app/current/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:36:1)
    at a.render (/var/app/current/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:41:338)
    at a.read (/var/app/current/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:40:436)
    at renderToString (/var/app/current/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:52:83)
    at renderPage (/var/app/current/node_modules/next/dist/server/render.js:319:26)
    at Function.getInitialProps (/var/app/current/node_modules/next/dist/server/document.js:65:25)
    at _callee$ (/var/app/current/node_modules/next/dist/lib/utils.js:86:30)
    at tryCatch (/var/app/current/node_modules/regenerator-runtime/runtime.js:62:40)
```

Solution: window should not be mandatory.

**Comment by xiaolin** [comment]
@rap2hpoutre Thanks for reporting. I totally overlooked `window` being put in an ssr lifecycle

**Comment by xiaolin** [comment]
fixed in `react-image-gallery@0.8.14`
